### ISSUE TITLE
Inconsistent timestamps in ROSE Data

### ISSUE THREAD

**Issue by eppfel** [secure-software-engineering/rose#263]
Comments a.o. record Zulu Datetime Strings, e.g. `2016-01-24T09:34:31.190Z`, but activity trackers record JS UNIX Timestamps, e.g. `1453628071190`.

Question from @YiranWangPARC14 : The Daylight saving time started on 3/13/16 2am in U.S., which was in the middle of my data collection. So some participants' data was before that time point, some after that time point; and for a few participants who participated during that week, some parts of their data (~ 5 days' data) were recorded before that particular time point and the rest (~ 2 days' data) after that time point. I wonder, should I manually offset any data that occurred before 3/13/16 2am by (-8) GMT, and data after 3/13/16 2am by (-7) GMT? Or, do the timestamps in the format of 2016-04-01T08:48:11.727Z in result1 and result2 already have the appropriate offset taken given the datetime compared to Daylight saving?

**Comment by iamolivinius** [comment]
Zulu, UTC and epoche time don' t care about daylight saving. If necessary one needs to convert the "timespamps" into local time.

**Comment by eppfel** [comment]
Proposal: use unix timestamps for the JSON export for less data and further handling. Human readable String could be exported in the csv files.
@apoller ?

**Comment by eppfel** [comment]
2nd propsal: unix timestamp and readable string as two fields in the json file.

**Comment by eppfel** [comment]
We also have to think about studies performed in multiple time zones and how this affects the ux of the researcher.

**Comment by iamolivinius** [comment]
As I remember we settled no Unix time stamps and decorate the exports with a human readable date format?!

**Comment by eppfel** [comment]
As I remember @apoller voted for storing both, for easier debugging. But I would not want to bloat the database.  
TL;DR: Yes

**Comment by eppfel** [comment]
Exported data file still has no intelligible Timestamp. Should add UTC String

**Comment by eppfel** [comment]
@apoller 
To ISOString or toString()? Or other format?

```JS
t.toString()
'Tue Jun 06 2017 12:40:28 GMT+0200 (CEST)'
t.toISOString()
'2017-06-06T10:40:28.396Z'
```

**Comment by eppfel** [comment]
183549ba8309f5bedf2629885840e5b85c4df080
